gprofiler/metadata/__init__.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CompletedProcess
+from threading import Event
+
+from granulate_utils.linux.ns import get_process_nspid, run_in_ns
+from psutil import Process
+
+from gprofiler.utils import run_process
+
+
+def get_exe_version(
+    process: Process,
+    stop_event: Event = None,
+    get_version_timeout: int = None,
```

Comment:
These 2 should be required - let's remove the default value for them, so that you cannot call this function without passing them (I don't want it to be called without passing both of them)